File: 8-1/2/2.c
#include <stdio.h>

int main(void){
    int a,b,c;
    scanf("%d %d %d",&a,&b,&c);
    if (a>b && a>c && b>c){
        printf("max: %d\n",a);
        printf("min: %d",c);
    }
    else if (a>b && a>c && c>b){
        printf("max: %d\n",a);
        printf("min: %d",b);
    }
    else if (a>b && c>a && c>b){
        printf("max: %d\n",c);
        printf("min: %d",b);
    }
    else if (b>a && c>a && c>b){
        printf("max: %d\n",c);
        printf("min: %d",a);
    }
    else if (b>a && a>c && b>c){
        printf("max: %d\n",b);
        printf("min: %d",c);
    }
    else{
        printf("max: %d\n",b);
        printf("min: %d",a);

    }
    return 0;
}

File: 6-2/2/2.py
import random


list=[]
for i in range(100):
    k=random.randint(1,1001)
    list.append(k)
    print(k,end=' ')


def findMax(ls):
    max=0
    for j in ls:
        if j >= max:
            max=j
    return max
def findMin(ls):
    min=1001
    for t in ls:
        if t <= min:
            min=t
    return min
print("")
print("")
print('max value: '+str(findMax(list)))
print("")
print('min value: '+str(findMin(list)))